### PR TITLE
Added Module

### PR DESCRIPTION
**Pull request by MarkWignall**
Added the module created at the end of Ex3.

### CODE REVIEW

File: modules/mdw/Animal/blib/libdoc/Animal.3pm [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+.        if !\nF==2 \{
+.            nr % 0
+.            nr F 2
+.        \}
+.    \}
+.\}
+.rr rF
+.\"
+.\" Accent mark definitions (@(#)ms.acc 1.5 88/02/08 SMI; from UCB 4.2).
+.\" Fear.  Run.  Save yourself.  No user-serviceable parts.
```
Comment (a-panayotopoulos) [inline modules/mdw/Animal/blib/libdoc/Animal.3pm]: :scream:

Comment (MarkWignall) [inline modules/mdw/Animal/blib/libdoc/Animal.3pm]: Is this an error or just ominous commenting?

Comment (a-panayotopoulos) [inline modules/mdw/Animal/blib/libdoc/Animal.3pm]: The line just jumped out at me. Must be part of the autogeneration.

Comment (MarkWignall) [inline modules/mdw/Animal/blib/libdoc/Animal.3pm]: Or maybe skynet likes me and sent me a personalized warning.

**Comment by codavex** [comment]
I've barely looked at the code I generated, so dunno if any of this is good, correct, nonsense. Seems to be okay though.

**Pull request merged**